retrieval/manager.go

```diff
@@ -54,8 +55,6 @@ type ScrapeManager struct {
 
 // Run starts background processing to handle target updates and reload the scraping loops.
 func (m *ScrapeManager) Run(tsets <-chan map[string][]*config.TargetGroup) error {
-	level.Info(m.logger).Log("msg", "Starting scrape manager...")
```

Comment:
Any reason this is removed?

Reply:
I didn't think it gives any useful info and no other manager/component has startup logs.
The TSDB has start up logs , but that one is useful as it also has another log `TSDB started`  when the initialisation has completed.